### ISSUE TITLE
Trouble with Schema generation

### ISSUE THREAD

**Issue by tagantroy** [gfx/Android-Orma#248]
boolean isFeatured(cursor.getLong(offset + 16) != 0);
                      ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:584: error: not a statement
    boolean isFeatured(cursor.getLong(offset + 16) != 0);
                                                   ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:584: error: ';' expected
    boolean isFeatured(cursor.getLong(offset + 16) != 0);
                                                       ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:585: error: ';' expected
    boolean isPopular(cursor.getLong(offset + 17) != 0);
                     ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:585: error: not a statement
    boolean isPopular(cursor.getLong(offset + 17) != 0);
                                                  ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:585: error: ';' expected
    boolean isPopular(cursor.getLong(offset + 17) != 0);
                                                      ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:586: error: ';' expected
    boolean isRecent(cursor.getLong(offset + 18) != 0);
                    ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:586: error: not a statement
    boolean isRecent(cursor.getLong(offset + 18) != 0);
                                                 ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:586: error: ';' expected
    boolean isRecent(cursor.getLong(offset + 18) != 0);
                                                     ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:587: error: ';' expected
    boolean isNearby(cursor.getLong(offset + 19) != 0);
                    ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:587: error: not a statement
    boolean isNearby(cursor.getLong(offset + 19) != 0);
                                                 ^
/Users/ivanbalaksha/Documents/ilitasoft/panoramaAndroidApp/app/build/generated/source/kapt/debug/com/ilitasoft/panorama/data/entity/Panorama_Schema.java:587: error: ';' expected
    boolean isNearby(cursor.getLong(offset + 19) != 0);
                                                     ^
12 errors

my class:

```kotlin
@Table
class Panorama {
    @PrimaryKey val id: Int
    @Column val title: String
    @Column val folder: String
    @Column val photography: String
    @Column val views: Int
    @Column val countryId: Int
    @Column val continentId: Int
    @Column val regionId: Int
    @Column val cityId: Int
    @Column val rating: Int
    @Column val commercial: Int
    @Column val favourite: Int
    @Column(indexed = true) val ltd: Float
    @Column(indexed = true) val lgt: Float
    @Column val athDelta: Float
    @Column val previewSmall: String
    @Column val previewBig: String
    @Column(indexed = true) val isFeatured: Boolean
    @Column(indexed = true) val isPopular: Boolean
    @Column(indexed = true) val isRecent: Boolean
    @Column(indexed = true) val isNearby: Boolean
    @Column val user: User?


    constructor(@Setter("id") id: Int,
                @Setter("title") title: String,
                @Setter("folder") folder: String,
                @Setter("photography") photography: String,
                @Setter("views") views: Int,
                @Setter("countryId") countryId: Int,
                @Setter("continentId") continentId: Int,
                @Setter("regionId") regionId: Int,
                @Setter("cityId") cityId: Int,
                @Setter("rating") rating: Int,
                @Setter("commercial") commercial: Int,
                @Setter("favourite") favourite: Int,
                @Setter("ltd") ltd: Float,
                @Setter("lgt") lgt: Float,
                @Setter("athDelta") athDelta: Float,
                @Setter("previewSmall") previewSmall: String,
                @Setter("previewBig") previewBig: String,
                @Setter("isFeatured") isFeatured: Boolean,
                @Setter("isPopular") isPopular: Boolean,
                @Setter("isRecent") isRecent: Boolean,
                @Setter("isNearby") isNearby: Boolean,
                @Setter("user") user: User?) {
        this.id = id
        this.title = title
        this.folder = folder
        this.photography = photography
        this.views = views
        this.countryId = countryId
        this.continentId = continentId
        this.regionId = regionId
        this.cityId = cityId
        this.rating = rating
        this.commercial = commercial
        this.favourite = favourite
        this.ltd = ltd
        this.lgt = lgt
        this.athDelta = athDelta
        this.previewSmall = previewSmall
        this.previewBig = previewBig
        this.isFeatured = isFeatured
        this.isPopular = isPopular
        this.isRecent = isRecent
        this.isNearby = isNearby
        this.user = user
    }
}
```

**Comment by gfx** [comment]
Seems bugs related to Kotlin.

What's your Kotlin version?

**Comment by tagantroy** [comment]
ext.kotlin_version = '1.0.1-2'

**Comment by gfx** [comment]
Fixed this issue & released v2.4.3. Can you check it out?
